tests/e2e/specs/layout.cy.js

```diff
@@ -20,8 +20,8 @@ describe('Default layout', () => {
     // visit any page first, so that we create the window.app reference
     cy.visit('/#/workflows/one')
     cy
-      .get('.v-alert')
-      .should('not.exist')
+      .get('.v-snack__wrapper')
+      .should('not.visible')
```

Comment:
Interesting that `be` (i.e. `not.be.visible`) is apparently optional

Reply:
Wasn't it you telling me it was syntactic sugar?

Reply:
I'm not too bothered